staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -257,12 +257,6 @@ type ObjectMeta struct {
 	// +patchStrategy=merge
 	Finalizers []string `json:"finalizers,omitempty" patchStrategy:"merge" protobuf:"bytes,14,rep,name=finalizers"`
 
-	// The name of the cluster which the object belongs to.
-	// This is used to distinguish resources with same name and namespace in different clusters.
-	// This field is not set anywhere right now and apiserver is going to ignore it if set in create or update request.
-	// +optional
-	ClusterName string `json:"clusterName,omitempty" protobuf:"bytes,15,opt,name=clusterName"`
```

Comment:
leave a tombstone comment that proto tag 15 and json name `clusterName` were previously used and can't be re-added with an incompatible type

we've done this in the past by commenting out a field definition:
```go
	// Fields is tombstoned to show why 5 is a reserved protobuf tag.
	//Fields *Fields `json:"fields,omitempty" protobuf:"bytes,5,opt,name=fields,casttype=Fields"`
```

Reply:
I will put it at the end of the struct to ensure no weird triggering of the "second before" comment block in the docs generator...